graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -218,8 +250,9 @@ def _rename_name_and_add_to_record(self, node):
                   NameNode.
 
         Returns:
-            Node object, identical to the input node, except with possibly a new name. If the
-            name was not changed, the returned object is the exact same object as the input
+            Node object or REMOVE. If it returns REMOVE, then the type was suppressed. Otherwise, it
```

Comment:
I'd make a note about what `REMOVE` is here, because that wasn't obvious to me. It's important for the reader to know that this is a constant defined by the GraphQL library, and not something this library invents.